paper-dropdown-menu.html

```diff
@@ -74,6 +77,12 @@
       text-align: left;
       cursor: pointer;
 
+      /* NOTE(cdata): Both values are needed, since some phones require the
+       * value to be `transparent`.
+       */
+      -webkit-tap-highlight-color: rgba(0,0,0,0);
+      -webkit-tap-highlight-color: transparent;
+
```

Comment:
This patch is orthogonal to the rest of this PR, but I figured I would sneak it in.